library/psa_crypto.c

```diff
@@ -1099,6 +1118,9 @@ psa_status_t psa_destroy_key(mbedtls_svc_key_id_t key)
 
 #if defined(MBEDTLS_PSA_CRYPTO_STORAGE_C)
     if (!PSA_KEY_LIFETIME_IS_VOLATILE(slot->attr.lifetime)) {
+        /* Destroy the copy of the persistent key from memory.
```

Comment:
Minor: Memory to me says non-volatile memory, maybe writing storage or non-volatile memory would be less ambiguous.

```suggestion
        /* Destroy the copy of the persistent key from storage.
```